Jolt/Physics/PhysicsSystem.h

```diff
@@ -114,7 +114,7 @@ class PhysicsSystem : public NonCopyable
 	static bool					sDrawMotionQualityLinearCast;								///< Draw debug info for objects that perform continuous collision detection through the linear cast motion quality
 
 	/// Draw the state of the bodies (debugging purposes)
-	void						DrawBodies(const BodyManager::DrawSettings &inSettings, DebugRenderer *inRenderer) { mBodyManager.Draw(inSettings, mPhysicsSettings, inRenderer); }
+	void						DrawBodies(const BodyManager::DrawSettings &inSettings, DebugRenderer *inRenderer, const BodyDrawFilter *inBodyFilter = nullptr) { mBodyManager.Draw(inSettings, mPhysicsSettings, inRenderer); }
```

Comment:
You forgot to pass inBodyFilter to the body manager.

Reply:
Oops, how embarassing! Because of the way our project is set up I can't commit my changes from the code I tested this in so I had to manually re-apply them to my Github checkout, and I only tested the case of not passing in a filter there... I have to test more thoroughly next time.